<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.flashtract.modules.invoice.api.rest.InvoiceWebApiTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2020-12-02T20:14:12" hostname="danielMac.local" time="0.027">
  <properties/>
  <testcase name="getInvoiceByInvoiceNumberTestException()" classname="com.flashtract.modules.invoice.api.rest.InvoiceWebApiTest" time="0.024"/>
  <testcase name="getInvoiceByInvoiceNumberTest()" classname="com.flashtract.modules.invoice.api.rest.InvoiceWebApiTest" time="0.002"/>
  <system-out><![CDATA[2020-12-02 15:14:12.821  INFO 16332 --- [    Test worker] c.f.m.invoice.api.rest.InvoiceWebApi     : web-api: Searching for all contracts
2020-12-02 15:14:12.822 ERROR 16332 --- [    Test worker] c.f.m.invoice.api.rest.InvoiceWebApi     : web-api: There was an error searching contracts
2020-12-02 15:14:12.833  INFO 16332 --- [    Test worker] c.f.m.invoice.api.rest.InvoiceWebApi     : web-api: Searching for all contracts
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
